### PR TITLE
gradual link hover and style change in index.html

### PR DESCRIPTION
**Pull request by sheenachhabradev**
https://github.com/Real-Dev-Squad/website-welcome/issues/23
@ankushdharkar  Please review and suggest.

### CODE REVIEW

File: css/common/main.css [hunk 1]
```diff
@@ -13,8 +13,10 @@ a {
 
 a:focus, a:hover {
   border-bottom: 1px solid;
-  background: #100170;
-  color: white;
+  background:white;
+  color:#2929a3;
+  transition-delay:0.2s;
```
Comment (ankushdharkar) [inline css/common/main.css:18]: Don't we also need a transition timing for the colors @sheenachhabradev?

Comment (sheenachhabradev) [reply css/common/main.css]: Added transition property and timing-function.

File: faq.html [hunk 2]
```diff
@@ -44,25 +48,54 @@ <h3 class="answer">As of now we're having 4 projects. Every project and the corr
...
+         	   vQestionset[vCounter].style.display = "block";
+			}
+			else
+			{
+			   console.log(vQuestions[0].innerText);
+			   vQestionset[vCounter].style.display = "none";
+			}
+		}
+	}
+  </script>
```
Comment (ankushdharkar) [inline faq.html:98]: @sheenachhabradev Does this code belong to this PR? It seems from the title that this PR was supposed to do visual & UX changes only

Can we have this fixed please?

File: faq.html [hunk 3]
```diff
@@ -44,25 +48,54 @@ <h3 class="answer">As of now we're having 4 projects. Every project and the corr
...
+			str= vQuestions[0].innerText;
+			if (str.toUpperCase().indexOf(filter) > -1)
+			{
+			   console.log(vQuestions[0].innerText);
+         	   vQestionset[vCounter].style.display = "block";
+			}
+			else
+			{
+			   console.log(vQuestions[0].innerText);
+			   vQestionset[vCounter].style.display = "none";
```
Comment (tejasberi) [inline faq.html:94]: i could see couple of console.logs in the JS. Need to remove console.logs.

File: faq.html [hunk 4]
```diff
@@ -44,25 +48,54 @@ <h3 class="answer">As of now we're having 4 projects. Every project and the corr
...
-
+	</div>
   </section>
   
 
   <footer>
     <p class="info-repo">The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank" rel="noopener noreferrer">open source repository</a></p>
   </footer>
-
+	<script>
```
Comment (tejasberi) [inline faq.html:71]: you can move this script to a .JS file and attach the JS file in the DOM. Having scripts in the HTML is not recommended unless unavoidable.

File: faq.html (hunk 2 above)
Comment (sheenachhabradev) [reply faq.html]: @ankushdharkar  Reverted the changes for other features.

File: faq.html (hunk 4 above)
Comment (sheenachhabradev) [reply faq.html]: @tejasberi  This is WIP . working on these . 
https://github.com/Real-Dev-Squad/website-welcome/pull/34

File: faq.html (hunk 2 above)
Comment (sheenachhabradev) [reply faq.html]: @ankushdharkar  any suggestions on this ?

**Pull request merged**